README.md

```diff
@@ -654,11 +665,43 @@ Possible statistics operation labels are
 Each operation section contains of different collectors
 for success calls and error (both error throw and `nil, err`)
 returns. `count` is total requests count since instance start
-or stats restart. `latency` is average time of requests execution,
+or stats restart. `latency` is 0.99 quantile of request execution
+time if `metrics` driver used, otherwise `latency` is total average.
 `time` is total time of requests execution.
 
+In `metrics` registry statistics are stored as `tnt_crud_stats` metrics
```

Comment:
Probably it is worth adding a link to documentation about monitoring
https://www.tarantool.io/en/doc/latest/book/monitoring/

Reply:
Added a couple of link to monitoring documentation